assets/schema_input.json

```diff
@@ -24,6 +24,88 @@
+                "maximum": 1
+            },
+            "normsizediff": {
+                "type": "number",
+                "errorMessage": "normsizediff is a svbenchmark parameter. Has to be between 0-1. Disallow matches if alternate alleles have normalized size difference greater than normsizediff (default 0.2)",
+                "meta": ["normsizediff"],
+                "default": 0.2,
+                "minimum": 0,
+                "maximum": 1
+            },
+            "maxdist": {
+                "type": "integer",
+                "errorMessage": "maxdist is a svbenchmark parameter. Disallow matches if positions of two variants are more than maxdist bases from each other (default 100,000).",
+                "meta": ["maxdist"],
+                "default": 100000
+            },
+            "pctsize": {
+                "type": "number",
+                "errorMessage": "pctsize is a truvari parameter. Has to be between 0-1. Ratio of min(base_size, comp_size)/max(base_size, comp_size).",
+                "meta": ["pctsize"],
+                "default": 0.7,
+                "minimum": 0,
+                "maximum": 1
+            },
+            "pctseq": {
+                "type": "number",
+                "errorMessage": "pctseq is a truvari parameter. Has to be between 0-1. Edit distance ratio between the REF/ALT haplotype sequences of base and comparison call. turn it off (0) for no sequence comparison.",
+                "meta": ["pctseq"],
+                "default": 0.7,
+                "minimum": 0,
+                "maximum": 1
+            },
+            "pctovl": {
+                "type": "number",
+                "errorMessage": "pctovl is a truvari parameter. Has to be between 0-1. Ratio of two calls' (overlapping bases)/(longest span)",
+                "meta": ["pctovl"],
+                "default": 0,
+                "minimum": 0,
+                "maximum": 1
+            },
+            "refdist": {
+                "type": "integer",
+                "errorMessage": "refdist is a truvari parameter. Maximum distance comparison calls must be within from base call's start/end ",
+                "meta": ["refdist"],
+                "default": 500
+            },
+            "chunksize": {
+                "type": "integer",
+                "errorMessage": "chunksize is a truvari parameter. Create chunks of all calls overlapping within ±`--chunksize` basepairs",
+                "meta": ["chunksize"],
+                "default": 500
+            },
+            "pick": {
+                "type": "string",
+                "default": "ac",
+                "anyOf": [
+                    {
+                        "dependentRequired": ["single"]
+                    },
+                    {
+                        "dependentRequired": ["ac"]
+                    },
+                    {
+                        "dependentRequired": ["multi"]
+                    }
+                ],
```

Comment:
```suggestion
                "enum": ["single", "ac", "multi"],
```

I think you actually want this, right? (`dependentRequired` can be used to set a requirement for other values if this value is set, while `enum` means that the value given to this input should be one of these)

Reply:
ach I see! That is what I was looking for!